.lighthouserc.json

```diff
@@ -0,0 +1,169 @@
+                    "uses-text-compression": ["warn", {"minScore": 0}], 
+                    "uses-http2": ["error", {"minScore": 0}], 
+                    "errors-in-console": ["error", {"minScore": 0}] 
+                }
+                },
+                {
+                "matchingUrlPattern": "http://[^/]+/contact",
+                "assertions": {
+                    "render-blocking-resources": ["warn", {"minScore": 0}], 
+                    "unminified-css": ["warn", {"minScore": 0}],  
+                    "unminified-javascript": ["warn", {"minScore": 0}], 
+                    "unused-css-rules": ["warn", {"minScore": 0}], 
+                    "uses-text-compression": ["warn", {"minScore": 0}], 
+                    "uses-http2": ["error", {"minScore": 0}], 
+                    "errors-in-console": ["error", {"minScore": 0}] 
+                }
+                },
+                {
+                "matchingUrlPattern": "http://[^/]+/about",
+                "assertions": {
+                    "render-blocking-resources": ["warn", {"minScore": 0}], 
+                    "unminified-css": ["warn", {"minScore": 0}],  
+                    "unminified-javascript": ["warn", {"minScore": 0}], 
+                    "unused-css-rules": ["warn", {"minScore": 0}], 
+                    "uses-text-compression": ["warn", {"minScore": 0}], 
+                    "uses-http2": ["error", {"minScore": 0}], 
+                    "errors-in-console": ["error", {"minScore": 0}] 
+                }
+                },
+                {
+                "matchingUrlPattern": "http://[^/]+/terms",
+                "assertions": {
+                    "render-blocking-resources": ["warn", {"minScore": 0}], 
+                    "unminified-css": ["warn", {"minScore": 0}],  
+                    "unminified-javascript": ["warn", {"minScore": 0}], 
+                    "unused-css-rules": ["warn", {"minScore": 0}], 
+                    "uses-text-compression": ["warn", {"minScore": 0}], 
+                    "uses-http2": ["error", {"minScore": 0}], 
+                    "errors-in-console": ["error", {"minScore": 0}] 
+                }
+                },
+                {
+                "matchingUrlPattern": "http://[^/]+/thanks",
+                "assertions": {
+                    "render-blocking-resources": ["warn", {"minScore": 0}], 
+                    "unminified-css": ["warn", {"minScore": 0}],  
+                    "unminified-javascript": ["warn", {"minScore": 0}], 
+                    "unused-css-rules": ["warn", {"minScore": 0}], 
+                    "uses-text-compression": ["warn", {"minScore": 0}], 
+                    "uses-http2": ["error", {"minScore": 0}], 
+                    "errors-in-console": ["error", {"minScore": 0}] 
+                }
+            }
+        ]
+        },
+        "upload": {
+            "target": "temporary-public-storage"
+          }
+
+    }
+}
```

Comment:
I have prepared the config here https://gist.github.com/vojtechjelinek/072054e99cd8ec5c57fa0e076cfc02e4 please use that one as there were a lot of changes and we need to get this in quickly. We can talk about the changes in the Speed team call.

Reply:
Done! I made increased the time for startServerReadyTimeout": to 250000, I was having issues with timing out when running the server script with only 150000.

Also running `python -m scripts.start --no_browser --prod_env` changes the   "DEV_MODE": to false in the  `constant.ts`
file for me. How should I handle this?

Reply:
Let's not care about this for now, I assume that @jameesjohn will change the script a bit when adding nginx anyway so he might tacle this in his PR.